### DISCUSSION TITLE
Unable to establish a connection to query-engine-node-api library. It seems that the current architecture x64 is not supported, or that libc is missing from the system.

### DISCUSSION THREAD

**Discussion by gholiveira29** [prisma/prisma#17570] [Q&A]
Ao executar o comando NPX PRISMA GENERATE, apresenta o seguinte erro : 
Error: Unable to establish a connection to query-engine-node-api library. It seems that the current architecture x64 is not supported, or that libc is missing from the system.
Details: Unable to require(`C:\Users\guide\OneDrive\Documentos\habities\server\node_modules\prisma\query_engine-windows.dll.node`)    
 Não foi possível encontrar o módulo especificado.
\\?\C:\Users\guide\OneDrive\Documentos\habities\server\node_modules\prisma\query_engine-windows.dll.node
[Context: getDmmf]

Poderiam me ajudar ?

**Comment by nurul3101** [comment]
Hello @gholiveira29 👋 

What version of Node are you using?
And do you have OpenSSL installed? If yes, which version?

**Reply by gholiveira29** [reply]
node 18.13.0 OpenSSL I don't have installed

**Reply by gholiveira29** [reply]
@nurul3101 
Is there any solution for this case?

**Reply by nurul3101** [reply]
@gholiveira29 Can you execute `openssl version` command in your windows machine and check if you get a version in the output?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
